CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## v2.1.0
+
+### Bug fixes and improvements
+
+* Use defaults for logFileDirectoryPath and logFileRegex that work for Greengrass components that print to standard output (stdout) and standard error (stderr). 
+* Correctly route traffic through a configured network proxy when uploading logs to CloudWatch Logs. 
+* Correctly handle colon characters (:) in log stream names. CloudWatch Logs log stream names don't support colons. 
+* Correctly form log stream names for things with multiple target thing groups 
```

Comment:
I think this should say something like `Remove thing group names from log streams for simplification`

Reply:
or just `Simplified log stream name construction`

Reply:
addressed